packages/tokens/package.json

```diff
@@ -31,16 +31,15 @@
     ],
     "scripts": {
         "clean:dist": "pnpm dlx rimraf dist",
-        "build": "node --loader ts-node/esm src/style-dictionary/build.ts"
+        "build": "tsx src/style-dictionary/build.ts"
     },
     "devDependencies": {
         "@types/node": "20.9.0",
         "@workleap/eslint-plugin": "3.0.0",
         "@workleap/tsup-configs": "3.0.1",
         "@workleap/typescript-configs": "3.0.2",
         "style-dictionary": "3.9.0",
-        "ts-node": "10.9.1",
-        "tsup": "7.2.0",
+        "tsx": "4.1.2",
```

Comment:
What's the strategy here @alexasselin008  we've gone from `tsup` to `ts-node` to `tsx`?

Reply:
tsup is still used to build libraries.

To execute node scripts: 
1- we had tsup that generated an intermediate build, that we would execute after with node. It was taking a long time
2- i remember i used ts-node in the past to skip the intermediate build, so i tried it and it worked, and faster than the 2 step build. However, i used to use ts-node with cjs code, with esm, we have experimental warning + a weird synthax `node --loader ts-node/esm`
3- By researching a bit further, i saw that `tsx` was recommended for TypeScript node file that were running ESM code. So now after the 3rd change i feel confident that we're in the right place for this project.

Sorry for changing it rapidly in a couple of weeks

Reply:
No problem, I just had to understand the approach, because it's a long way from what we've defined in tech design.